### PR TITLE
Stylize scrollbar

### PR DESCRIPTION
**Pull request by paradigm**

### CODE REVIEW

File: markdown/bedrock.css [hunk 1]
```diff
@@ -26,7 +26,8 @@ html{
 	background:#141414;
 	font-family:'Open Sans', sans-serif;
 	-webkit-text-size-adjust:100%;
-	-ms-text-size-adjust:100%
+	-ms-text-size-adjust:100%;
+	padding: 1em
```
Comment (Bobbbay) [inline markdown/bedrock.css:30]: Double check the output of this, adding a padding on all sides is generally dangerous, and `1em` (by default) is usually alot :)

Comment (paradigm) [reply markdown/bedrock.css]: Updated to `padding-bottom: 3px`.  Let me know if that's better.

**Comment by Bobbbay** [comment]
@paradigm does this fix #37?

**Comment by paradigm** [comment]
Yes, yes it is.  While it's not surprising, I wasn't aware of Github's ability to make those kinds of associations.  Neat.

**Comment by Bobbbay** [comment]
@paradigm yeah, just saying fixes #37 is super useful (and fun) to have. 

Also, I noticed you're manually deleting branches. I'd recommend turning on auto delete on merge :) I dunno if you're doing it manually on purpose, but here ya go:

1. Navigate to main page of the repository and click on Settings.
2. Under "Merge button", you can select or unselect "Automatically delete head branches" option.

Also, I've been lurking around some more, will open a couple issues here! Cheers!

**Pull request merged**